engine/common/follower/engine.go

```diff
@@ -390,7 +425,7 @@ func (e *Engine) processBlockAndDescendants(ctx context.Context, proposal *messa
 	// check whether the block is a valid extension of the chain.
 	// it only checks the block header, since checking block body is expensive.
 	// The full block check is done by the consensus participants.
-	err := e.state.Extend(ctx, block)
+	err = e.state.Extend(ctx, block)
```

Comment:
```suggestion
	// TODO: CAUTION we write a block to disk, without validating its payload yet. This is vulnerable to malicious primaries. 
	err = e.state.Extend(ctx, block)
```

Reply:
added in https://github.com/onflow/flow-go/pull/3408